### DISCUSSION TITLE
To get live information about the connection I made a conky script that looks like this..

### DISCUSSION THREAD

**Discussion by krolingo** [pgj/freebsd-wifibox#48] [Show and tell]
![Screenshot from 2023-02-01 01-57-45](https://user-images.githubusercontent.com/214111/215973635-dfbebd7c-4270-4df7-9759-936f3e64f57a.png)

**Comment by krolingo** [comment]
`#!/bin/sh

while true; do
  wpa_cli status | grep "ip_address" | awk '{ sub("ip_address=", "", $1); print $1 }' > /tmp/wifibox_ip
  bssid=$(wpa_cli status | grep "bssid=" | awk '{ sub("bssid=", "", $1); print $1 }')
  printf "%s" "$bssid" > /tmp/wifibox_bssid
  ~/bin/wifi_strength.sh > /tmp/wifibox_signal_strenght
  sleep 5
done
`

**Comment by pgj** [comment]
Oh, that is nice!  @krolingo would you like to me to include this script?

**Comment by krolingo** [comment]
> Oh, that is nice! @krolingo would you like to me to include this script?

I posted about it on my blog: https://eldapper.wordpress.com/2023/01/30/my-freebsd-dashboard/

You decide to do with it whatever you want ;)

And thank you for Wifibox! it has made my FreeBSD experience 100% better

**Comment by pgj** [comment]
I still could not decide how to carry on with this, but I will leave this issue open as a pointer for others for now.

**Comment by pgj** [comment]
@krolingo I have converted your issue to a discussion.  I believe that is probably a better way to communicate its contents.
